tests/sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__scalar_percentiles_v1/test_minimal/data.py

```diff
@@ -0,0 +1,84 @@
+"""Testing data for query."""
+from pathlib import Path
+from uuid import uuid4
+
+import yaml
+
+ROOT = Path(__file__).parent
+
+SUBMISSION_DATE = "2020-10-01"
+APP_BUILD_ID = "2020100100"
+
+# TODO: counts are wrong here?
+CLIENTS_SCALAR_AGGREGATES = [
+    {
+        "client_id": str(uuid4()),
+        "ping_type": "*",
+        "os": "*",
+        "app_version": 84,
+        "app_build_id": "*",
+        "channel": "*",
+        "scalar_aggregates": [
+            {
+                "metric": "places_manager_write_query_count",
+                "metric_type": "counter",
+                "key": "",
+                "agg_type": "count",
+                "value": 4.0,
+            },
+        ],
+    },
+    {
+        "client_id": str(uuid4()),
+        "ping_type": "*",
+        "os": "*",
+        "app_version": 84,
+        "app_build_id": "*",
+        "channel": "*",
+        "scalar_aggregates": [
+            {
+                "metric": "places_manager_write_query_count",
+                "metric_type": "counter",
+                "key": "",
+                "agg_type": "count",
+                "value": 8.0,
+            },
+        ],
+    },
+]
+
+# TODO: the total user count is very wrong
```

Comment:
This is the most egregious error, there should be at most 2 users.

Reply:
Looking at the query it looks like this is because of the cross join with the `*` combos.  In the final query you should be able to do `COUNT(DISTINCT(client_id)) AS total_users`

Reply:
Thanks for catching this, this fixes the user counts.